### PR TITLE
Footer: adding legalCopy prop for GDPR

### PR DESCRIPTION
**Pull request by jmhammons**
#### Related issues
Fixes https://meetup.atlassian.net/browse/WC-216

#### Description

Adds a `legalCopy` prop to the `Footer` comp for GDPR compliance.

#### Screenshots (if applicable)

See ticket, but here is the idea...

gdpr-footer-copy

### CODE REVIEW

File: src/Footer.jsx [hunk 1]
```diff
@@ -187,6 +190,13 @@ export const Footer = ({
 							separator="·"
```
Comment (jmhammons) [inline src/Footer.jsx]: Would be nice if I can add the separator at medium, probably.

File: src/Footer.jsx [hunk 2]
```diff
@@ -20,6 +20,8 @@ import withMatchMedia from './utils/components/withMatchMedia';
 
 import { getSocialLinks } from './utils/getSocialLinks';
 
+export const defaultLegalCopy = 'Meetup is a wholly owned subsidiary of WeWork Companies, Inc.';
```
Comment (jmhammons) [inline src/Footer.jsx]: Is this the right way to go in this repo -- english defaults here and 'apps' pass in translated strings?

Comment (mperrotti) [reply src/Footer.jsx]: This is fine, but I'd recommend just defining this in the story and passing it in from there.

Comment (jmhammons) [reply src/Footer.jsx]: Hey @mperrotti -- just getting back today. So is the recommendation to define the english string in the story and the component separately? Is it okay that the story and component might get out of sync copy-wise. I'm fine to change it if that's how it's done here, just trying to understand the reasoning.

Comment (mperrotti) [reply src/Footer.jsx]: I'm suggesting not to define anything in the component. Just put the `legalCopy` prop in place, and then pass `legalCopy="Meetup is a wholly...`

File: src/Footer.jsx [hunk 3]
```diff
@@ -183,10 +184,17 @@ export const Footer = ({
 				<Bounds>
 					<div className="padding--all">
 						<InlineBlockList
-							className="text--small align--center atMedium_align--left"
+							className="text--small align--center atMedium_align--left footerList-legal"
 							separator="·"
 							items={[
 								<span>{`© Meetup ${new Date().getFullYear()}`}</span>,
+								(legalCopy && <span>{legalCopy}</span>),
+							].filter(item => item)}
```
Comment (jmhammons) [inline src/Footer.jsx]: I _think_ I need this, so we don't have an unwanted `<li/>` if a consumer app doesn't pass `legalCopy` in, but will double-check.

**Pull request merged**